pkg/server/app/app.go

```diff
@@ -328,6 +330,33 @@ func checkForProtectedEnvVars(evsNames []string) error {
 	return nil
 }
 
+func (ops *AppOperations) List(user *storage.User) ([]*List, error) {
+	teams, err := ops.tops.ListByUser(user.Email)
+	if err != nil {
+		return nil, err
+	}
+	lists := make([]*List, 0)
```

Comment:
💅 "lists" is a bit confuse name, what do you thing about `result` or `response` for example? 💅